homeassistant/components/light/__init__.py

```diff
@@ -32,6 +32,13 @@
 
 ENTITY_ID_FORMAT = DOMAIN + ".{}"
 
+# Configuration for possible effects
+CONF_EFFECT_LIST = "effect_list"
+
+PLATFORM_SCHEMA.extend({
```

Comment:
I think that should be a part of platform which effect is sopported like other platform/component does. Look at climate mode list or mediaplayer source selection. Move this to mqtt platform in this way.